src/parsec_client/operations/README.md

```diff
@@ -133,4 +134,8 @@ stability of their contracts or abstract interfaces.**
 - [PrepareKeyAttestation](prepare_key_attestation.md)
 - [AttestKey](attest_key.md)
 
+### Capability discovery
+
+- [CanDoCrypto](can_do_crypto.md)
```

Comment:
I think it's worth to add the same "EXPERIMENTAL" notice as above. We can remove it once the operation is supported everywhere and we all agree